README.md

```diff
@@ -36,14 +41,72 @@ git submodule update
 For more information about (working with) submodules, see:
 https://git-scm.com/book/en/v2/Git-Tools-Submodules
 
-The EESSI specific settings can be found in group_vars/all.yml, and in templates we added a template for a Squid configation for the local proxy servers; this file is not included in the Galaxy repository.
+### Configuration
+
+The EESSI specific settings can be found in group_vars/all.yml, and in templates we added a template for a Squid configation for the local proxy servers; this file is not included in the Galaxy repository. For all playbooks you will also need to have an appropriate Ansible hosts file; see the supplied hosts.example for the structure and host groups that you need for these playbooks.
+
+## Running the playbooks
 
-In order to run one of these playbooks, you will have to use a hosts file that includes at least the group of nodes for which you are running a playbook. An example of the file that shows the correct structure can be found in hosts.example.
-The playbooks can then be run as follows:
+In general, all the playbooks can be run like this:
 ```
 ansible-playbook -i hosts -b <name of playbook>.yml
 ```
-where -b means become, i.e. run with sudo. If this requires a password, include -K:
+where -i allows you to specify the pah to your hosts file, and -b means become, i.e. run with sudo. If this requires a password, include -K, which will ask for the sudo password when running the playbook:
 ```
 ansible-playbook -i hosts -b -K <name of playbook>.yml
 ```
+
+Before you run any of the commands below, make sure that you updated the file group_vars/all.yml and include the new/extra URLs of any server you want to change/add (e.g. add your Stratum 1).
+
+### Stratum 0
+First install the Stratum 0 server:
+```
+ansible-playbook -i hosts -b -K stratum0.yml
+```
+
+Then install the files for the configuration repository:
+```
+ansible-playbook -i hosts -b -K stratum0-deploy-cvmfs-config.yml
+```
```

Comment:
Should we add a note here to clarify that this is only required for testing purposes, since there's only a single central Stratum-0 in the EESSI project?

Should we mention the Stratum-0 URL here?

Reply:
Added a note here. Didn't include the URL: you don't really need (to know) this, unless you want to set up a Stratum 1, but then you can find it in the all.yml.